Big-Spender/readme.md

```diff
@@ -124,7 +124,11 @@ CREATE YOUR QUERY HERE
 **You:** Sure thing. To confirm, the date is August 19, 2021, the transaction number is 38104091, the supplier invoice number is 3780119655, the supplier is 'Dell', the expense type is 'Hardware' and the expense area is 'IT'. Here's the query for that:
 
 ```sql
-INSERT YOUR QUERIES HERE
+INSERT INTO expense_types (expense_type) VALUES ('Hardware');
+INSERT INTO expense_areas (expense_area) VALUES ('IT');
+INSERT INTO suppliers (supplier) VALUES ('Dell');
+INSERT INTO spends (expense_type_id, expense_area_id, supplier_id, date, transaction_no, supplier_inv_no, description, amount) VALUES (83, 91, 131,'2021-08-19', 38104091, 3780119655, 'Computer Hardware Dell', 32000);
```

Comment:
yep, looks good. assuming the values for the foreign key ids are those which were added in the tables above, right?

Reply:
yes, id from insert tables = foreign keys